packages/shared/package.json

```diff
@@ -14,7 +14,10 @@
     "clean": "rimraf dist coverage",
     "test": "jest"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@mikro-orm/core": "^5.7.14",
```

Comment:
I don't think we need this dependency; can you remove it and rebuild everything to see if anything breaks?